win32.mak

```diff
@@ -9,17 +9,20 @@ BUILD=release
 OS=windows
 DMD=$(DMD_DIR)\generated\$(OS)\$(BUILD)\32\dmd
 
+# Used for running MASM assembler on .asm files
 CC=dmc
+
 MAKE=make
 HOST_DMD=dmd
 
 DOCDIR=doc
 IMPDIR=import
 
+# For compiling D and ImportC files
 DFLAGS=-m32omf -conf= -O -release -preview=dip1000 -preview=fieldwise -preview=dtorfields -inline -w -Isrc -Iimport
+# For unittest build
 UDFLAGS=-m32omf -conf= -O -release -preview=dip1000 -preview=fieldwise -w -Isrc -Iimport
-DDOCFLAGS=-conf= -c -w -o- -Isrc -Iimport -version=CoreDdoc
```

Comment:
Isn't this a symptom of documentation only being built on Linux/POSIX? Shouldn't the make files instead be synced up so doc generation recipes are (re?)added to the windows makefiles?

Reply:
If it's ever needed, it should be added then. Currently, it is unused and just a distraction.

Reply:
IMO, the Makefiles should all be merged to a single one for all platforms in the long-term, using GNU make. LDC uses GNU make on Windows for the druntime tests (`test` subdir; not using the win64.mak files for those - Windows extras have been added to the 'Posix' makefiles and can be upstreamed), so it's definitely an option.